lib/wickedbase.pm

```diff
@@ -234,6 +234,31 @@ sub get_test_result {
     }
 }
 
+sub get_wicked_logs {
+    my ($self, $prefix) = @_;
+    my $dir_name = $self->{name} . '_' . $prefix;
+    my $logs_dir = "/tmp/$dir_name";
+    record_info('Logs', "Collecting logs in $logs_dir");
+    script_run("mkdir -p $logs_dir");
+    script_run("date +'%Y-%m-%d %T.%6N' > $logs_dir/date");
+    script_run("journalctl -b -o short-precise|tail -n +2|grep wicked > $logs_dir/journalctl.log");
+    script_run("wicked ifstatus --verbose all > $logs_dir/wicked_ifstatus.log 2>&1");
+    script_run("wicked show-config > $logs_dir/wicked_config.log 2>&1");
```

Comment:
totally ok to have this thing here but we may drop duplicate logic which we already have in before_test.pm

Reply:
I think I missed that function, sorry. Will remove it.